tests/unit/draw/DefaultDrawer.spec.js

```diff
@@ -44,648 +49,263 @@ describe('Test Class: DefaultDrawer()', () => {
-          definition: { model: 'CustomModel' },
+          depth: 1,
+          data: {},
+          height: 0,
+          parent: null,
+          value: 1,
         }],
-        definition: { model: 'bad' },
       }];
-      drawer.draw('parent');
-
-      expect(drawer.d3.select).toBeCalledTimes(4);
-      expect(drawer.d3.selectAll).toBeCalledTimes(4);
-      expect(drawer.d3.data).toBeCalledTimes(2);
-      expect(actionCallBack.data[0].data({ id: 'id0' })).toEqual('id0');
-      expect(actionCallBack.data[1].data({ id: 'id1' })).toEqual('id1');
-      expect(drawer.initializeComponents).toBeCalledTimes(2);
-      expect(drawer.setSelectionAction).toBeCalledTimes(5);
-      expect(drawer.initializeActionMenu).toBeCalledTimes(2);
-      expect(drawer.__drawModels).toBeCalledTimes(2);
-
-      expect(drawer.d3.attr).toBeCalledTimes(8);
-      expect(actionCallBack.attr[0].name)
-        .toEqual('id');
-      expect(actionCallBack.attr[0].callback({ id: 'idTest0' })).toEqual('idTest0');
-      expect(actionCallBack.attr[1].name).toEqual('class');
-      expect(actionCallBack.attr[1].callback({ definition: { model: 'test0', type: 'type0' } }))
-        .toEqual('parent type0 component component-test0 ');
-      expect(actionCallBack.attr[2].name).toEqual('id');
-      expect(actionCallBack.attr[2].callback({ id: 'idTest1' })).toEqual('idTest1');
-      expect(actionCallBack.attr[3].name).toEqual('class');
-      expect(actionCallBack.attr[4].name).toEqual('x');
-      expect(actionCallBack.attr[4].callback({ drawOption: { x: 1 } })).toEqual(1);
-      expect(actionCallBack.attr[5].name).toEqual('y');
-      expect(actionCallBack.attr[5].callback({ drawOption: { y: 2 } })).toEqual(2);
-      expect(actionCallBack.attr[6].name).toEqual('x');
-      expect(actionCallBack.attr[6].callback({ drawOption: { x: 3 } })).toEqual(3);
-      expect(actionCallBack.attr[7].name).toEqual('y');
-      expect(actionCallBack.attr[7].callback({ drawOption: { y: 4 } })).toEqual(4);
-
-      expect(drawer.d3.html).toBeCalledTimes(2);
-      expect(actionCallBack.html[0].callback({ definition: { model: 'test0' } })).toEqual('testModel0');
-      expect(actionCallBack.html[1].callback({ definition: { model: 'test1' } })).toEqual('testModel1');
     });
-  });
-
-  describe('Test method: initializeComponents', () => {
-    let drawer;
-    let d3Element;
-    let components;
-    let component00;
-    let component01;
-    let component02;
-    let actionCallBack;
-    beforeEach(() => {
-      actionCallBack = { attr: [], html: {} };
-      drawer = new DefaultDrawer(new DefaultData());
-      drawer.d3 = mockD3(jest);
-      d3Element = mockD3(jest);
-      drawer.pack = jest.fn().mockReturnValue({
-        width: 1,
-        height: 1,
-        items: [
-          { item: { id: 'a' }, x: 1, y: 2 },
-          { item: { id: 'b' }, x: 3, y: 4 },
-          { item: { id: 'c' }, x: 5, y: 6 },
-        ],
+    it('should set x0,y0,x1 and y1 in each item', () => {
```

Comment:
each describe and it should have empty line before unles is the first.
for all `should` rename to `Should`

Reply:
done